src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -12,31 +10,14 @@ public TestingContext(IMessageCreator messageCreator = null) : base(messageCreat
 
         public TestingContext(IMessageCreator messageCreator, TimeoutMessage<object>[] timeoutMessages) : base(messageCreator)
         {
-            this.timeoutMessages = new ConcurrentQueue<TimeoutMessage<object>>(timeoutMessages);
+            this.previousTimeouts = timeoutMessages;
         }
 
-        public TimeoutMessage<object>[] TimeoutMessages => timeoutMessages.ToArray();
```

Comment:
Are these changes because of the changes to the testing fakes? With this code being here, does that mean the change to the testing fakes actually wasn't needed, or does it just make more sense to it to be there instead?